lets-plot-compose/src/androidMain/kotlin/org/jetbrains/letsPlot/skia/compose/android/PlotViewContainer.kt

```diff
@@ -7,30 +7,43 @@ package org.jetbrains.letsPlot.skia.compose.android
 
 import android.annotation.SuppressLint
 import android.content.Context
-import android.view.View
 import android.widget.RelativeLayout
 import org.jetbrains.letsPlot.Figure
 import org.jetbrains.letsPlot.commons.geometry.DoubleRectangle
 import org.jetbrains.letsPlot.commons.geometry.DoubleVector
 import org.jetbrains.letsPlot.commons.geometry.Vector
 import org.jetbrains.letsPlot.commons.registration.Disposable
+import org.jetbrains.letsPlot.commons.registration.Registration
 import org.jetbrains.letsPlot.core.util.MonolithicCommon
 import org.jetbrains.letsPlot.core.util.PlotSizeUtil
 import org.jetbrains.letsPlot.intern.toSpec
 import org.jetbrains.letsPlot.skia.android.MonolithicSkiaAndroid
+import org.jetbrains.letsPlot.skia.android.view.SvgPanel
 import org.jetbrains.letsPlot.skia.compose.util.NaiveLogger
 
 private val LOG = NaiveLogger("PlotViewContainer")
 
+// If false then new SvgPanel() is created for each update.
+val reusePlotSvgPanel = true
```

Comment:
Lets clean up the code and have just one version, i.e. "reuse".

Reply:
Done